jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/RadioButtonTests.java

```diff
@@ -1,53 +1,66 @@
 
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.tools.Timer;
 import io.github.epam.TestsInit;
+import org.hamcrest.Matchers;
+import org.testng.annotations.BeforeTest;
 import org.testng.annotations.Test;
 
-import static io.github.com.StaticSite.radioDefaultPage;
-import static io.github.com.StaticSite.radioButtonFrame;
-import static org.hamcrest.Matchers.containsString;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static io.github.com.pages.inputs.RadioButtonPage.*;
+import static io.github.com.MaterialNavigator.openSection;
+import static io.github.com.StaticSite.radioButtonPage;
 
 
 public class RadioButtonTests extends TestsInit {
+    static private final ArrayList<String> labels = new ArrayList<>(Arrays.asList("First", "Second", "Third", "Disabled"));
+    static private final ArrayList<String> classes = new ArrayList<>(Arrays.asList("Top", "Start", "Bottom"));
+    static private final ArrayList<String> messages = new ArrayList<>(Arrays.asList("You got it!", "Sorry, wrong answer!"));
+    static private final Timer timer = new Timer(2000L);
+
+    @BeforeTest()
+    public void beforeTest() {
+        openSection("Radio");
+        radioButtonPage.isOpened();
+    }
 
     @Test
-    public void defaultRadioButtonTest() {
-        radioDefaultPage.open();
-
-        radioButtonFrame.firstRadioButton.is().enabled();
-        radioButtonFrame.secondRadioButton.is().enabled();
-        radioButtonFrame.thirdRadioButton.is().enabled();
-        radioButtonFrame.disabledRadioButton.is().disabled();
-
-        radioButtonFrame.firstRadioButton.is().unchecked();
-        radioButtonFrame.secondRadioButton.is().unchecked();
-        radioButtonFrame.thirdRadioButton.is().unchecked();
-        radioButtonFrame.disabledRadioButton.is().unchecked();
-
-        radioButtonFrame.firstRadioButton.toggle();
-        radioButtonFrame.firstRadioButton.is().checked();
-        radioButtonFrame.secondRadioButton.is().unchecked();
-        radioButtonFrame.thirdRadioButton.is().unchecked();
-        radioButtonFrame.disabledRadioButton.is().unchecked();
-        radioButtonFrame.textField.is().text(containsString("First"));
-
-        radioButtonFrame.secondRadioButton.toggle();
-        radioButtonFrame.firstRadioButton.is().unchecked();
-        radioButtonFrame.secondRadioButton.is().checked();
-        radioButtonFrame.thirdRadioButton.is().unchecked();
-        radioButtonFrame.disabledRadioButton.is().unchecked();
-        radioButtonFrame.textField.is().text(containsString("Second"));
-
-        radioButtonFrame.firstRadioButton.is().unchecked();
-        radioButtonFrame.secondRadioButton.is().checked();
-        radioButtonFrame.thirdRadioButton.is().unchecked();
-        radioButtonFrame.disabledRadioButton.is().unchecked();
+    public void simpleRadioTest() {
+        for (int i = 1; i <= 4; i++) {
+            Button button = simpleRadioButtons.get(i);
```

Comment:
rename this variable to
currentRadioButton

Reply:
done